### PR TITLE
Migrate to updated @graknlabs_dependencies repo

### PR DESCRIPTION
**Pull request by vmax**
## What is the goal of this PR?

Use updated versions of dependencies (with some of them downgraded) in Grakn Core

## What are the changes implemented in this PR?

Override certain dependency versions to avoid compatibility issues

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -220,6 +220,13 @@ load("@graknlabs_verification//dependencies/maven:artifacts.bzl", graknlabs_veri
 ##############################
 load("@graknlabs_grakn_core//dependencies/maven:artifacts.bzl", graknlabs_grakn_core_artifacts = "artifacts")
 
+# Override libraries conflicting with versions defined in @graknlabs_dependencies
+GRAKN_CORE_OVERRIDES = {
```
Comment (flyingsilverfin) [inline WORKSPACE:224]: doesn't look too bad!

File: .bazelrc [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # what is defined in this section will be applied at all times
-build --workspace_status_command bin/workspace_status.sh --show_result=0 --show_progress=no --incompatible_strict_action_env
-test --show_result=0 --show_progress=no --incompatible_strict_action_env
-run --show_result=0 --show_progress=no --incompatible_strict_action_env
+build --workspace_status_command bin/workspace_status.sh --incompatible_strict_action_env
+test --show_result=0 --incompatible_strict_action_env
+run --show_result=0 --incompatible_strict_action_env
```
Comment (haikalpribadi) [inline .bazelrc:4]: What are these changes for, @vmax ?

File: dependencies/graknlabs/dependencies.bzl [hunk 3]
```diff
@@ -20,8 +20,8 @@ load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 def graknlabs_dependencies():
     git_repository(
         name = "graknlabs_dependencies",
-        remote = "https://github.com/graknlabs/dependencies",
-        commit = "c921c9468468249764eebafa9940a487f577c3f8",  # sync-marker: do not remove this comment, this is used for sync-dependencies by @graknlabs_dependencies
+        remote = "https://github.com/vmax/graknlabs-dependencies",
+        commit = "ec9462cc70ce7366385ac12f83d20906523acd96",  # sync-marker: do not remove this comment, this is used for sync-dependencies by @graknlabs_dependencies
```
Comment (haikalpribadi) [inline dependencies/graknlabs/dependencies.bzl:24]: Is there a PR on `@graknlabs_dependencies` for this already? That should be merged first before this PR goes in.

**Comment by haikalpribadi** [comment]
Is the test failure legitimate, @vmax ?

File: dependencies/graknlabs/dependencies.bzl (hunk 3 above)
Comment (vmax) [reply dependencies/graknlabs/dependencies.bzl:24]: @haikalpribadi yes (graknlabs/dependencies#157), we'll make sure it gets merged before this one

File: .bazelrc (hunk 2 above)
Comment (vmax) [reply .bazelrc:4]: @haikalpribadi it's hard to debug stuff when progress is not shown; Core is the only repo that had these flags set so I think it's appropriate to remove them: `--show_result=0`, `--show_progress=no`

**Comment by vmax** [comment]
@haikalpribadi 

I'm rerunning the test; @flyingsilverfin says it's flaky and unlike other failures that we've seen it doesn't seem to be related to dependencies per se

File: .bazelrc (hunk 2 above)
Comment (haikalpribadi) [reply .bazelrc:4]: right, makes sense. Should we do this across all repos, @vmax ?

**Pull request merged**